src/util/states.ts

```diff
@@ -520,6 +520,7 @@ export function blurSeries(
         }
         const view = api.getViewOfComponentModel(componentModel);
         if (view && view.blurSeries) {
+            getComponentStates(componentModel).isBlured = true;
```

Comment:
The logic here is for the markPoint/markLine/markArea component to blur the makers elements in the `blurredSeries`. It's not blur the component itself. 

Marking this `componentModel` to `isBlured` means the component model needs to be blurred. It's not exactly the same. 

What's more correct is providing a method  like`unblurSeries` in the `MarkerView` that do the opposite operation and call it in the `allLeaveBlur` methods.